docs/rules/comma-style.md

```diff
@@ -1,4 +1,4 @@
-# Comma style (comma-style)
+# Enforce One True Comma Style (comma-style)
```

Comment:
Ah, I see now. Was curious what "One True Comma Style" was :P I agree with you, @platinumazure